bookings/components/booking-filters.yaml

```diff
@@ -0,0 +1,464 @@
+  in: query
+  required: false
+  schema:
+    type: boolean
+  description: >-
+    Filters on bookings which are private hire.
+query:
+  name: query
+  in: query
+  required: false
+  example: '@example.com'
+  schema:
+    type: string
+  description: >-
+    Filter on bookings which have a `full_name`, `email`, `company` or
+    `phone` similar to the search query.
+start_time:
+  name: start_time
+  in: query
+  required: false
+  schema:
+    type: string
+  example: '13:15'
+  description: >-
+    Filters on bookings which have the given start time. This filter can also 
+    accept a range of times, for example `12:00 TO 15:30` will filter bookings 
+    starting between 12pm and 3:30pm.
+status_changed_date:
+  name: status_changed_date
+  in: query
+  required: false
+  schema:
+    type: string
+    format: date
+  example: '2019-01-01'
+  description: >-
+    Filters on bookings where the status was changed on the given date.
+status_changed_date_from:
+  name: status_changed_date_from
+  in: query
+  required: false
+  schema:
+    type: string
+    format: date
+  example: '2019-01-01'
+  description: >-
+    Filters on bookings where the status was changed on or after the given date.
+status_changed_date_to:
+  name: status_changed_date_to
+  in: query
+  required: false
+  schema:
+    type: string
+    format: date
+  example: '2019-03-01'
+  description: >-
+    Filters on bookings where the status was changed on or before the given date.
+status:
+  name: status
+  in: query
+  required: false
+  explode: false
+  example: in_progress
+  description: >-
+    Filters on bookings that have the given status. If a comma-separated list is 
+    passed, bookings that have any of the given statuses will be returned.
+  schema:
+    type: array
+    items:
+      type: string
+      enum:
+        - new
+        - in_progress
+        - complete
+        - rejected
+        - deleted
+        - lost
+time:
+  name: time
+  in: query
+  required: false
+  description: >-
+    Filters on bookings that take place between the given time range, taking into 
+    account both the start time and the end time of the booking.
+  example: '12:00 TO 13:00'
+  schema:
+    type: string
+type:
+  name: type
+  in: query
+  required: false
+  explode: false
+  description: >-
+    Filters on bookings that have the given booking type ID. If a comma-separated 
+    list is passed, bookings with any of the given booking type IDs will be returned.
+  example: 5cab0bcbfcf59f7fd26bedd9
+  schema:
+    type: array
+    items:
+      type: string
+customer_id:
+  name: customer_id
+  in: query
+  required: false
+  explode: false
+  example: 5cab0c245ca3d2d984a4b938
+  schema:
+    type: array
+    items:
+      type: string
+  description: >-
+    Filters on bookings that are associated with the given customer ID. If a comma-separated 
+    list is passed, bookings associated with any of the given customer IDs will be returned.
+venue_group:
```

Comment:
Only DMN users can view bookings from more than one venue group at a time, so I wonder if it's worth omitting this from the docs?